README.md

```diff
@@ -1,5 +1,7 @@
 # Nuxt Toolbox Template
 
+This is a [NuxtJS](https://nuxtjs.org/) v2 project. It is a reference on how to integrate commonly used features within Netlify for Nuxt.js.
```

Comment:
```suggestion
![netlify + nuxt logo](https://res.cloudinary.com/dzkoxrsdj/image/upload/v1645592629/CleanShot_2022-02-22_at_22.47.16_2x_ltqti0.png)

This is a [NuxtJS](https://nuxtjs.org/) v2 project. It is a reference on how to integrate commonly used features within Netlify for Nuxt.js.
```

i have this placeholder until i find the real design resources that i know exist somewhere 🤔